OWNERS.yml

```diff
@@ -0,0 +1,76 @@
+---
+# Node.js Project Codeowners
+
+# 1. Codeowners must always be teams, never individuals
+# 2. Each codeowner team should contain at least one TSC member
+# 3. PRs touching any code with a codeowner must be signed off by at least one
+#    person on the code owner team.
+
+./OWNERS.yml: ["@nodejs/tsc"]
+
+# net
+
+./deps/cares: ["@nodejs/net"]
+./doc/api/dns.md: ["@nodejs/net"]
+./doc/api/dgram.md: ["@nodejs/net"]
+./doc/api/net.md: ["@nodejs/net"]
+./lib/dgram.js: ["@nodejs/net"]
+./lib/dns.js: ["@nodejs/net"]
+./lib/net.js: ["@nodejs/net", "@nodejs/quic"]
+./lib/internal/dgram.js: ["@nodejs/net"]
+./lib/internal/dns/*: ["@nodejs/net"]
+./lib/internal/net.js: ["@nodejs/net"]
+./lib/internal/socket_list.js: ["@nodejs/net"]
+./lib/internal/js_stream_socket.js: ["@nodejs/net"]
+./src/cares_wrap.h: ["@nodejs/net"]
+./src/connect_wrap.*: ["@nodejs/net"]
+./src/connection_wrap.*: ["@nodejs/net"]
+./src/node_sockaddr*: ["@nodejs/net"]
+./src/tcp_wrap.*: ["@nodejs/net"]
+./src/udp_wrap.*: ["@nodejs/net"]
+
+# tls/crypto
+
+./lib/internal/crypto/*: ["@nodejs/crypto"]
+./lib/internal/tls.js: ["@nodejs/crypto", "@nodejs/net"]
+./lib/crypto.js: ["@nodejs/crypto"]
+./lib/tls.js: ["@nodejs/crypto", "@nodejs/net"]
+./src/node_crypto*: ["@nodejs/crypto"]
+./src/node_crypto_common*: ["@nodejs/crypto", "@nodejs/quic"]
+
+# http
+
+./deps/llhttp/*: ["@nodejs/http", "@nodejs/net"]
+./doc/api/http.md: ["@nodejs/http", "@nodejs/net"]
+./lib/_http_*: ["@nodejs/http", "@nodejs/net"]
+./lib/http.js: ["@nodejs/http", "@nodejs/net"]
+./lib/https.js: ["@nodejs/crypto", "@nodejs/net", "@nodejs/http"]
+./src/node_http_common*: ["@nodejs/http", "@nodejs/http2", "@nodejs/quic", "@nodejs/net"]
+./src/node_http_parser.cc: ["@nodejs/http", "@nodejs/net"]
+
+# http2
+
+./deps/nghttp2/*: ["@nodejs/http2", "@nodejs/net"]
+./doc/api/http2.md: ["@nodejs/http", "@nodejs/http2", "@nodejs/net"]
+./lib/http2.js: ["@nodejs/http2", "@nodejs/net"]
+./lib/internal/http2/*: ["@nodejs/http2", "@nodejs/net"]
+./src/node_http2*: ["@nodejs/http2", "@nodejs/net"]
+./src/node_mem*: ["@nodejs/http2"]
```

Comment:
If we are aiming for compatibility with CODEOWNERS here, the CODEOWNERS format does not like the leading `.` at the start of the path.

Reply:
As mentioned above, the file was converted from our own CODEOWNERS. I don't mind removing the leading `./` from it though.